### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by weller7**
# Title - docs: Update README documentation

Solves # 14

## Description
Added the information we reviewed to the readme with the markdown syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,76 @@
...
+3. Install dependencies using npm install.
+4. Set up the backend server and database.
+5. Run the frontend application using npm start.
+6. Access the application through your web browser.
+7. Sign up or log in to start using the Kanban board for managing your job hunt.
+
+# Contribute
+Contributions to the project are welcome! If you'd like to contribute, please follow these guidelines:
+
+(Link to contribute .env)
```
Comment (alcpereira) [inline README.md:66]:
```suggestion
[Contributing guide](CONTRIBUTING.md)
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,76 @@
...
+
+# License
+This project is licensed under the MIT License
+
+# To go further
 - Login (Discord OAuth)
 - Add/rename columns
 - List of companies (Reviews)
 - Analytics
```
Comment (alcpereira) [inline README.md:75]: I think we can delete those, they will be present in the `docs/features.md` file (see #13)

File: README.md [hunk 3]
```diff
@@ -1,33 +1,76 @@
...
+│   ├── client<br>
+│   │   └── assets<br>
+│   └── server<br>
+│       └── assets<br>
+├── coverage<br>
+├── docs<br>
+├── public<br>
+└── utils<br>
+    ├── mermaid<br>
+    └── seed<br>
```
Comment (alcpereira) [inline README.md:52]: You can use the triple backtick (\`\`\`) around this block to have a nice formatting (and remove the `<br>`)

I'd also remove: 
- `Card` and `Column`, they are implementation details
- `client` and `server` folders with their `assets`, also an implementation detail

No need for more details for now, as we have an Issue to add more details to this structure (see #22)

File: README.md [hunk 4]
```diff
@@ -1,33 +1,76 @@
...
+- **Task Management:** Users can create tasks/cards within each column to represent job applications, networking opportunities, follow-ups, or any other relevant activities. These cards can include details such as company name, job title, application status, contact information, and deadlines.
+
+- **Drag-and-Drop Functionality:** The board will support drag-and-drop functionality, allowing users to easily move cards between columns to reflect changes in the status of their job search activities. For example, moving a card from "Applied" to "Interview Scheduled" when a job interview is confirmed.
+
+- **Deadline Reminders:** The board will provide the option to set deadlines for tasks, with optional reminders to ensure users stay on track and meet important application deadlines.
+
+
+# Technologies used
+- **Frontend:** Tailwind CSS, Typescript (React.js  
+- **Backend:** Remix (SSR React) + Vite + Express, Turso (for database),Storybook, Vitest + React Testing Library + Playwright
```
Comment (alcpereira) [inline README.md:32]:
```suggestion
- **Language:** TypeScript
- **Framework:** Remix (React.js) with Vite and Express
- **Frontend libraries:** Tailwind CSS, Storybook
- **Database:** Turso with libSQL (in memory, and hosted as a service)
- **Testing:**  Vitest, React Testing Library, and Playwright
```

File: README.md [hunk 5]
```diff
@@ -1,33 +1,76 @@
 # 100Hunters
-
 Group project
```
Comment (alcpereira) [inline README.md:2]:
```suggestion
```

**Review by alcpereira — APPROVED** [review]
LGTM 👍 

Just some small modifications

**Review by alcpereira — APPROVED** [review]

**Review by JoeDravarol — APPROVED** [review]
LGTM 👍

**Pull request merged**